src/layout/Cell.js

```diff
@@ -41,7 +41,7 @@
         var context = this;
         var offsetMultiple;
 
-        var dragHandles = element.selectAll(".dragHandle").data(this._dragHandles, function (d) { return d; });
+        var dragHandles = element.selectAll("#"+this.id()+" > .dragHandle").data(this._dragHandles, function (d) { return d; });
```

Comment:
I think the **selectChildren** is worthwhile, but for completeness did you mean to suggest this?
```javascript
Widget.prototype.selectAll = function(selector) {
    return this.selectChildren(selector);
});
```